srtcore/core.cpp

```diff
@@ -5446,9 +5443,15 @@ void CUDT::bstats(CBytePerfMon* perf, bool clear)
    {
       if (m_pSndBuffer)
       {
-         //new>
          #ifdef SRT_ENABLE_SNDBUFSZ_MAVG
-         perf->pktSndBuf = m_pSndBuffer->getAvgBufSize(Ref(perf->byteSndBuf), Ref(perf->msSndBuf));
+         if (instantaneous 
+         ||  ((perf->pktSndBuf == 0x5AE) && (perf->msSndBuf ==0x5AE))){ //5AE: SRT Adapted Encoding: Preserve historical API to get instantaneous stats
```

Comment:
Could you please add more explanation in form of code comments on 0x5AE magic?
Also is this endian-safe?

Reply:
Before the API wa changed to add the instantaneous param, and because of lazy programming, the perf structure was used as and IN and OUT param to enable the "instantaneous" behavior. The value 0x5Ae is just a signature set in 2 field to lower the probability of accidental stack junk setting. I hope this comment will stay for history as much as a comment would be and save me from redoing anothe PR.

Reply:
If there's a chance for a singular value ("stack junk") in this structure, then maybe we need to somehow initialize it?

Reply:
I would vote for removing the magic if it is already deprecated. @jeandube let me know if you agree and I will push the change.